portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,9 +29,9 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-class submitComment{
+class comment{
```

Comment:
I missed this in #3, and I'm not a Java expert, but class names are always capitalized, as far as I know.

https://google.github.io/styleguide/javaguide.html#s5.2.2-class-names